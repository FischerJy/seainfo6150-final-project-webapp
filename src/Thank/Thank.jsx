import React from "react";
import thankyou from "../images/thankyou.jfif";

export default function Thank() {
  return (
    <div className="t-img-wrap main">
      <img src={thankyou} width="100%" height="100%" alt="thank you" />
    </div>
  );
}
