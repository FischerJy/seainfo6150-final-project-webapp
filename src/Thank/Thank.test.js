import React from "react";
import { render } from "@testing-library/react";
import Thank from "./Thank.jsx";
import { BrowserRouter } from "react-router-dom";



it("renders the Thank component correctly", () => {
    const { container } = render(<BrowserRouter>
        <Thank />
    </BrowserRouter>);
    expect(container).toMatchSnapshot();
});