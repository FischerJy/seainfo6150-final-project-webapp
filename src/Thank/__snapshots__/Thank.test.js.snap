// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Thank component correctly 1`] = `
<div>
  <div
    class="t-img-wrap main"
  >
    <img
      alt=""
      src="https://ss1.bdstatic.com/70cFvXSh_Q1YnxGkpoWK1HF6hhy/it/u=199378973,2076223274&fm=15&gp=0.jpg"
    />
  </div>
</div>
`;
